BLSQ/iaso
# WC2-110 workflows READ

List/Details view of workflows version for a specific beneficiary type.

## Self proof reading checklist


## Changes

New workflows domain only to READ


## How to test

Go to the `/admin/iaso/workflow/`
Create a new workflow linked to a entity type.
Go to the `/admin/iaso/workflowversion/`
Create multiple versions of the workflow you created.
You can add changes and followups too.
For Changes: Mapping should be something like 
`{"key":"value"}`
For Followups: condition is a jsonlogic expression (true is also working)

Test filters, sort, results, pagination, navigation


## Print screen / video
<img width="847" alt="Screenshot 2022-12-12 at 14 28 07" src="https://user-images.githubusercontent.com/12494624/207057786-77522192-03ec-4119-b44b-2a72062cc488.png">
<img width="831" alt="Screenshot 2022-12-12 at 14 32 14" src="https://user-images.githubusercontent.com/12494624/207057829-f6a7ff39-ed40-4d18-8433-e3e022d82b84.png">
<img width="1677" alt="Screenshot 2022-12-12 at 14 33 52" src="https://user-images.githubusercontent.com/12494624/207057849-2180732c-3c93-46d1-8b67-0f46a29884ea.png">
